README.md

```diff
@@ -85,6 +85,9 @@ After starting systemd-sysext.service (`sudo systemctl enable --now systemd-syse
 **Note**: An extension created this way will be linked against specific libraries on your system and will not work on other distributions.
 It also requires the previously mentioned libraries/dependencies at runtime to be installed in your system (the system extension does not carry these libraries).
 
+**Read-Only Filesystem**: If you're not on an immutable distro you may notice that `/usr/` and `/opt/` are read-only.
+this is caused by `systemd-sysext` being enabled, when you are done testing you can disable `systemd-sysext` (`sudo systemctl disable systemd-sysext`)
```

Comment:
You could add `--now` to invoke systemctl stop implicitly https://www.freedesktop.org/software/systemd/man/systemctl.html#disable%20UNIT%E2%80%A6